activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+*   Add `Arel::Nodes::Cte` for use in `WITH` expressions
```

Comment:
Should this be in the CHANGELOG? Arel is private so this isn't really a feature we should be promoting in the CHANGELOG since it is in theory not user facing.

Reply:
Ah -- I didn't consider that. I can open a PR to remove the changelog entry.